eclass/common-lisp-3.eclass

```diff
@@ -120,7 +120,8 @@ common-lisp-install-sources() {
 		if [[ -f ${path} ]] ; then
 			common-lisp-install-one-source ${fpredicate} "${path}" "$(dirname "${path}")"
 		elif [[ -d ${path} ]] ; then
-			common-lisp-install-sources -t ${ftype} $(find "${path}" -type f)
+			readarray -d '' FILES < <(find "${path}" -type f -print0)
```

Comment:
`FILES` is a local variable AFAICS? Then is should be declared as one, and the convention is to use lowercase for its name.

`|| die` is missing.

Reply:
Where would the `die` go? `readarray` won't fail (I don't think), would it go inside the `<(...)`? Previous subshelling of find didn't `die`, already checks that path is dir but not readable, so could fail there.

Next line with `common-lisp-install-sources` also `die`s properly in recursion, so shouldn't be needed there.